src/rules/prefer-strict-equal.ts

```diff
@@ -36,14 +37,19 @@ export default createRule({
           isParsedEqualityMatcherCall(matcher, EqualityMatcher.toEqual)
         ) {
           context.report({
-            fix: fixer => [
-              fixer.replaceText(
-                matcher.node.property,
-                EqualityMatcher.toStrictEqual,
-              ),
-            ],
             messageId: 'useToStrictEqual',
             node: matcher.node.property,
+            suggest: [
```

Comment:
add `suggestion: true` to the `meta.docs` field as well

Reply:
[We need `@typescript-eslint` to define the property first](https://github.com/typescript-eslint/typescript-eslint/pull/2033)

Reply:
`@ts-expect-error` in the meantime seems prudent 🙂

Reply:
For that we'd have to update typescript ;)

(I'll do it quickly on another branch)

Reply:
actually the PR got merged, so might as well wait until the release tomorrow & then update 🤷

Reply:
Also fyi using `@ts-expect-error` is a bad move here because it completely breaks inference, so TS won't error on things like invalid `messageId` values 😬